src/Slices/ServiceInventory/UI/Components/ForceStateAction.test.tsx

```diff
@@ -0,0 +1,187 @@
+import React from "react";
+import { fireEvent, screen, render, act } from "@testing-library/react";
+import { StoreProvider } from "easy-peasy";
+import { EnvironmentDetails, RemoteData } from "@/Core";
+import {
+  TriggerForceStateCommandManager,
+  CommandResolverImpl,
+  BaseApiHelper,
+  KeycloakAuthHelper,
+  getStoreInstance,
+} from "@/Data";
+import {
+  dependencies,
+  DynamicCommandManagerResolver,
+  MockEnvironmentModifier,
+  ServiceInstance,
+} from "@/Test";
+import { DependencyProvider, EnvironmentModifierImpl } from "@/UI/Dependency";
+import { ForceStateAction } from "./ForceStateAction";
+
+function setup() {
+  const commandManager = TriggerForceStateCommandManager(
+    new KeycloakAuthHelper(),
+    new BaseApiHelper()
+  );
+  return {
+    commandResolver: new CommandResolverImpl(
+      new DynamicCommandManagerResolver([commandManager])
+    ),
+  };
+}
+function setupComponent() {
+  const { commandResolver } = setup();
+  return {
+    component: (
+      <DependencyProvider
+        dependencies={{
+          ...dependencies,
+          commandResolver,
+          environmentModifier: new MockEnvironmentModifier(),
+        }}
+      >
+        <ForceStateAction
+          id={ServiceInstance.a.id}
+          instance_identity={
+            ServiceInstance.a.service_identity_attribute_value ??
+            ServiceInstance.a.id
+          }
+          service_entity={ServiceInstance.a.service_entity}
+          version={ServiceInstance.a.version}
+          targets={["up", "deleting"]}
+        />
+      </DependencyProvider>
+    ),
+  };
+}
+
+test("ForceStateAction dropdown doesn't takes environment halted status in account", async () => {
```

Comment:
Why are you using test instead of describe/it here ?

Reply:
changed